packages/openmrs/src/http.js

```diff
@@ -0,0 +1,43 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * State object
+ * @typedef {Object} OpenMRSOptions
```

Comment:
Is this a duplicate typedef? I don't actually know if this works, but can we re-use typedefs across files? 

Ideally `util.js` would define the set of options, and we reference the type in the docs here, rather than duplicating it.

Reply:
@josephjclark  I did discover a way to share `typedef` across files

Reply:
Wow great work!